src/nanoc/content/support.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Support
+---
+
+[Lightbend]: https://www.lightbend.com
+
+## Support
```

Comment:
Could you add link to http://www.scala-sbt.org/community.html#how-can-I-get-help and deduplicate it?

Reply:
I'm not a fan of the duplication too, but it (generally) looks like what's been done between https://www.lagomframework.com/support.html and https://www.lagomframework.com/get-involved.html.

What kind of deduplification did you mean? Because if you mean point this page to that, then it would be quite a bare page.

Reply:
Sorry, I meant to say adding link _from_ community.html, not _to_. 
Get involved should talk mostly about ppl helping sbt, and the new Support can talk about sbt helping ppl.

Reply:
Oh right. Sure sounds good. Will do.